### PR TITLE
Update Imperative to resolve line-break issues in the web-help

### PR DESCRIPTION
**Pull request by zFernand0**
https://github.com/zowe/imperative/issues/715

### CODE REVIEW

**Review by t1m0thyj — APPROVED** [review]
Thanks for adding the `lerna clean -y` to prevent unnecessary package-lock changes 🙂

**Comment by zFernand0** [comment]
> Thanks for adding the `lerna clean -y` to prevent unnecessary package-lock changes 🙂

I also added `npm i` at the end in order to update the top level pacakge-lock
Maybe I could've done `npm i --package-lock-only` ?

Also, forgot to update snapshots 😢

File: package.json [hunk 1]
```diff
@@ -16,7 +16,7 @@
     "lint": "eslint \"packages/**/*.ts\" \"**/__tests__/**/*.ts\"",
     "lint:packages": "eslint \"packages/**/*.ts\" --ignore-pattern \"**/__tests__/**/*.ts\"",
     "lint:tests": "eslint \"**/__tests__/**/*.ts\"",
-    "update:imperative": "npm i --save --save-exact @zowe/imperative@latest && syncpack fix-mismatches --dev --prod --filter @zowe/imperative",
+    "update:imperative": "lerna clean -y && npm i --save --save-exact @zowe/imperative@latest && syncpack fix-mismatches --dev --prod --filter @zowe/imperative && npm i",
```
Comment (t1m0thyj) [inline package.json:19]: On my machine it seems like `npm i --save --save-exact @zowe/imperative@latest` causes Imperative to get duped 😢 

Perhaps we want to try this instead?
`"update:imperative": "npm i --package-lock-only --no-package-lock --save-exact @zowe/imperative@latest && syncpack fix-mismatches --dev --prod --filter @zowe/imperative && npm i",`

Comment (zFernand0) [reply package.json]: thank you! 😋

**Comment by zFernand0** [comment]
Waiting on decision about possible breaking change 😢

**Comment by zFernand0** [comment]
> Waiting on decision about possible breaking change 😢

Good to go as a `patch` release 😋

**Review by awharn — APPROVED** [review]

**Pull request merged**